avocado/core/loader.py

```diff
@@ -44,6 +44,41 @@
 ALL = True
 
 
+def filter_test_tags(test_suite, filter_by_tags):
+    '''
+    The filtering mechanism is limited to instrumented tests with tags set
+    '''
```

Comment:
I'm all in favor of trying our best to stick to one style, and the prevalent is indeed `"""`.  I was justing pointing out that they're equal, if for some reason, there was some urban legend pointing otherwise! :)